File: src/Entity/Activite.php
<?php

namespace App\Entity;

use App\Repository\ActiviteRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=ActiviteRepository::class)
 */
class Activite
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $nom_activite;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $type_activite;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $prix_activite;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $nb_pers;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $date_debut;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $date_fin;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $fiche_descriptive;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNomActivite(): ?string
    {
        return $this->nom_activite;
    }

    public function setNomActivite(string $nom_activite): self
    {
        $this->nom_activite = $nom_activite;

        return $this;
    }

    public function getTypeActivite(): ?string
    {
        return $this->type_activite;
    }

    public function setTypeActivite(string $type_activite): self
    {
        $this->type_activite = $type_activite;

        return $this;
    }

    public function getPrixActivite(): ?string
    {
        return $this->prix_activite;
    }

    public function setPrixActivite(string $prix_activite): self
    {
        $this->prix_activite = $prix_activite;

        return $this;
    }

    public function getNbPers(): ?string
    {
        return $this->nb_pers;
    }

    public function setNbPers(string $nb_pers): self
    {
        $this->nb_pers = $nb_pers;

        return $this;
    }

    public function getDateDebut(): ?string
    {
        return $this->date_debut;
    }

    public function setDateDebut(string $date_debut): self
    {
        $this->date_debut = $date_debut;

        return $this;
    }

    public function getDateFin(): ?string
    {
        return $this->date_fin;
    }

    public function setDateFin(string $date_fin): self
    {
        $this->date_fin = $date_fin;

        return $this;
    }

    public function getFicheDescriptive(): ?string
    {
        return $this->fiche_descriptive;
    }

    public function setFicheDescriptive(string $fiche_descriptive): self
    {
        $this->fiche_descriptive = $fiche_descriptive;

        return $this;
    }
}

File: Gestion Loisir Desktop/src/Entities/Activite.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

/**
 *
 * @author Gaston
 */
public class Activite {
    
    
    private Integer  id ; 
    private String nom_activite ; 
    private String type_activite ; 
    private String prix_activite ; 
    private String nb_pers ; 
    private String date_debut ; 
    private String date_fin ; 
     private String fiche_descriptive ; 
     private String pays ;
     private String place ;

    public Integer getId() {
        return id;
    }

    public String getNom_activite() {
        return nom_activite;
    }

    public String getType_activite() {
        return type_activite;
    }

    public String getPrix_activite() {
        return prix_activite;
    }

    public String getNb_pers() {
        return nb_pers;
    }

    public String getDate_debut() {
        return date_debut;
    }

    public String getDate_fin() {
        return date_fin;
    }

    public String getFiche_descriptive() {
        return fiche_descriptive;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public void setNom_activite(String nom_activite) {
        this.nom_activite = nom_activite;
    }

    public void setType_activite(String type_activite) {
        this.type_activite = type_activite;
    }

    public void setPrix_activite(String prix_activite) {
        this.prix_activite = prix_activite;
    }

    public void setNb_pers(String nb_pers) {
        this.nb_pers = nb_pers;
    }

    public void setDate_debut(String date_debut) {
        this.date_debut = date_debut;
    }

    public void setDate_fin(String date_fin) {
        this.date_fin = date_fin;
    }

    public void setFiche_descriptive(String fiche_descriptive) {
        this.fiche_descriptive = fiche_descriptive;
    }

    public Activite(String nom_activite, String type_activite, String prix_activite, String nb_pers, String date_debut, String date_fin, String fiche_descriptive, String pays, String place) {
        this.nom_activite = nom_activite;
        this.type_activite = type_activite;
        this.prix_activite = prix_activite;
        this.nb_pers = nb_pers;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.fiche_descriptive = fiche_descriptive;
        this.pays = pays;
        this.place = place;
    }

  

    public Activite() {
    }

    public String getPlace() {
        return place;
    }

    public String getPays() {
        return pays;
    }

    public void setPays(String pays) {
        this.pays = pays;
    }

    public void setPlace(String place) {
        this.place = place;
    }

    
     
     
}
